tests/pipeline/test_iri_pipeline.py

```diff
@@ -0,0 +1,48 @@
+"""Test IRI methods as called from the pipeline."""
```

Comment:
Can you remove some of the other IRI tests because of this file, or merge them here?

Reply:
Hmm yes I was first thinking about this as well. But then felt like we should keep pipeline and the main functionalities separate. And therefore I would argue to also keep the tests of them separate, else it gets pretty diluted. Why would you say we should have them all in the pipeline file?

Reply:
Oh well I was thinking if you go through `pipeline` then you can test everything through there and don't have to write duplicates. But since the future of pipeline is uncertain then you can leave it for now